File: spotifyarch/src/main/java/com/timothy/spotifyarchitecture/retrofit/models/ErrorDetails.java
package com.timothy.spotifyarchitecture.retrofit.models;

import android.os.Parcel;
import android.os.Parcelable;

/**
 * <a href="https://developer.spotify.com/web-api/object-model/#error-object">Error object model</a>
 */
public class ErrorDetails {
    public int status;
    public String message;

    public ErrorDetails() {
    }
}

File: spotifyarch/src/main/java/com/timothy/spotifyarchitecture/retrofit/models/ErrorDetails.kt
package com.timothy.spotifyarchitecture.retrofit.models

import android.os.Parcel
import android.os.Parcelable

/**
 * [Error object model](https://developer.spotify.com/web-api/object-model/#error-object)
 */
class ErrorDetails {
    var status: Int = 0
    var message: String = ""
}
